client.go

```diff
@@ -420,6 +430,13 @@ func (client *Client) ProjectID() string {
 	return client.projectID
 }
 
+func (client *Client) Release() string {
+	client.mu.RLock()
+	defer client.mu.RUnlock()
+
+	return client.release
+}
+
```

Comment:
Nitpick: I prefer the getters to come in the same order as the variables on the client. Another in-place violation, but moving this to the bottom of the getters is what I'm looking for.